src/components/Pages/MainPage/MainPage.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { RenderResult, within } from '@testing-library/react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { renderWithProviders } from '../../../tests/tests'
+import MainPage from './MainPage'
+import { Props } from './MainPage.types'
+
+jest.mock('decentraland-ui/dist/components/Profile/Profile', () => {
+  const module = jest.requireActual('decentraland-ui/dist/components/Profile/Profile')
+  return {
+    ...module,
+    Profile: () => <div></div>
+  }
+})
+
+// Mock tab rendering as it is including semantic-ui Container components that is not working correctly in jest
+jest.mock('decentraland-ui/dist/components/Tabs/Tabs', () => {
+  const module = jest.requireActual('decentraland-ui/dist/components/Tabs/Tabs')
+  const Tabs = (props: { children: React.ReactNode }) => <div data-testid="main-tabs">{props.children}</div>
+  Tabs.Tab = module.Tabs.Tab
+  return {
+    ...module,
+    Tabs
+  }
+})
+
+function renderMainPage(props: Partial<Props> = {}) {
+  return renderWithProviders(
+    <MainPage
+      loggedInAddress="0xtest"
+      profileAddress="0xaddress"
+      isCreationsTabEnabled={false}
+      isAssetsTabEnabled={false}
+      isBlocked={false}
+      {...props}
+    />
+  )
+}
+
+let screen: RenderResult
+
+describe('when creations tab FF is enabled', () => {
```

Comment:
If I'm not wrong, the article might be missing:
```suggestion
describe('when the creations tab FF is enabled', () => {
```